.devcontainer/devcontainer.json

```diff
@@ -3,7 +3,7 @@
 {
 	"name": "Go",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/go:0-1.20-bullseye",
+	"image": "mcr.microsoft.com/devcontainers/go:1-1.22-bullseye",
```

Comment:
This image came from [Microsoft Artifact Registry](https://mcr.microsoft.com/en-us/product/devcontainers/go/tags).

<img width="1358" alt="image" src="https://github.com/cinar/indicator/assets/756988/724194c3-b0a3-4824-b87c-a5f580490555">